### PR TITLE
fix most lint issues

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[RN] Disable typescript linter and enable only esLint linter (10.01)](https://trello.com/c/xbCIyccb/1053-rn-fix-some-linter-issues)

**Comment by peterBrxwn** [comment]
[[RN] Disable typescript linter and enable only esLint linter (10.01)](https://trello.com/c/xbCIyccb/1053-rn-fix-some-linter-issues)

**Review by turtledreams — APPROVED** [review]

File: Countly.js [hunk 1]
```diff
@@ -1585,8 +1585,9 @@ Countly.showStarRating = function (callback) {
  * @param {callback listener} [ratingWidgetCallback] This parameter is optional.
  */
 Countly.presentRatingWidgetWithID = function (widgetId, closeButtonText, ratingWidgetCallback) {
```
Comment (arifBurakDemiray) [inline Countly.js:1587]: in function's doc, return is missing if it returns smthng. Message is returned but, does it intended thing?

File: Utils.js [hunk 2]
```diff
@@ -16,18 +16,20 @@ const DeviceIdType = {
...
+    case 'DS':
+        result = DeviceIdType.DEVELOPER_SUPPLIED;
+        break;
+    case 'TID':
+        result = DeviceIdType.TEMPORARY_ID;
+        break;
+    case 'SG':
+        result = DeviceIdType.SDK_GENERATED;
+        break;
+    default:
```
Comment (arifBurakDemiray) [inline Utils.js:28]: does adding default a linter fix?

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]
Maybe they are not related but they caught my attention

**Pull request merged**